sdk/ml/azure-ai-ml/azure/ai/ml/entities/_deployment/batch_deployment.py

```diff
@@ -110,6 +112,9 @@ def __init__(
         **kwargs,
     ) -> None:
 
+        self.deployment_type = kwargs.pop("type", "Model")
```

Comment:
We spec'd it s.t. we default to Model?

Reply:
I'm not sure about this. Taji worked on these changes according to the spec. So, it most likely is documented but if not we can document it too.